docs/source/posts/2022/2022-07-04-week3-shivam.rst

```diff
@@ -0,0 +1,42 @@
+Week 3 - Fixing fetcher, adding tests and docs
+==============================================
+
+.. post:: July 04 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- The first task for this week was to fix the glTF fetcher, We noticed that while running tests it reaches the API limit. So we decided to use a json file that contains the download URLs for all models.
+
+- Created a function to generate JSON files with download URLs for all models, Which can be found `here <https://github.com/xtanion/fury/blob/gltf-json-gen/fury/data/fetcher.py#L330>`_.
+
+- Modified the tests and ``fetcher.py`` to download using the json URLs and Merged the PR `#616 <https://github.com/fury-gl/fury/pull/616>`_.
```

Comment:
maybe `JSON`

Reply:
Either `Which` should be changed to lowercase or the comma to a full stop